test/clj_momo/lib/es/document_test.clj

```diff
@@ -288,11 +317,58 @@
     (is (= 3 (es-doc/count-docs conn "test_index" "test_mapping" {:ids {:values (range 3)}})))
     (es-index/delete! conn "test_index")))
 
-(deftest ^:integration query-test
+(deftest ^:integration scroll-test
   (let [sample-docs (mapv #(assoc {:_index "test_index"
                                    :_type "test_mapping"
                                    :foo :bar}
                                   :_id (str %))
+                          (range 100))
+        conn (es-conn/connect (th/get-es-config))
+        _ (es-index/delete! conn "test_index")
+        _ (es-index/create! conn "test_index" {})
+        _ (es-doc/bulk-create-doc conn sample-docs "true")
+        {batch-1 :data
+         {scroll-id-1 :scroll_id} :paging} (es-doc/query conn
+                                                         "test_index"
+                                                         "test_mapping"
+                                                         {:match_all {}}
+                                                         {:scroll "10s" :limit 40})
+       {batch-2 :data
+        {scroll-id-2 :scroll_id} :paging} (es-doc/scroll conn
+                                                         scroll-id-1
+                                                         {:scroll "10s"
+                                                          :full-hits? true})
```

Comment:
Why use `:full-hits` only here? If the situation is specific shouldn't we make a specific test for that?